styles.css

```diff
@@ -0,0 +1,1514 @@
+    display: flex;
+  }
+
+  .css p {
+    margin-left: 16px;
+    width: 98px;
+    height: 24px;
+    left: 12px;
+    top: 76px;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    display: flex;
+    align-items: center;
+    color: #253858;
+  }
+
+  .java {
+    margin-bottom: 20px;
+    width: 160%;
+    height: 50px;
+    padding: 108px;
+    background-color: #f7f7f9;
+    font-size: 48px;
+    color: rgb(207, 187, 11);
+    border-radius: 20px;
+    display: flex;
+  }
+
+  .java p {
+    margin-left: 16px;
+    width: 98px;
+    height: 24px;
+    left: 12px;
+    top: 76px;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    display: flex;
+    align-items: center;
+    color: #253858;
+  }
+
+  .dropmenu1 {
+    display: none;
+  }
+
+  .dropmenu1 div {
+    padding: 12px;
+    margin-right: 12px;
+    left: 2%;
+  }
+
+  .languajes:hover .dropmenu1 {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .boton {
+    border: none;
+  }
+
+  .boton p {
+    width: 85%;
+    padding-bottom: 45px;
+    border-bottom: 1px solid #dfe1e6;
+    position: absolute;
+    font-family: Poppins, sans-serif;
+    left: 8%;
+    height: 24px;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+    display: flex;
+    align-items: center;
+  }
+
+  .arrow {
+    position: absolute;
+    width: 100%;
+  }
+
+  .arrows p {
+    position: relative;
+    font-size: 25px;
+    z-index: 3;
+    margin-left: 170%;
+    bottom: 22px;
+    color: #acb7c3;
+  }
+
+  .languajes:hover .boton p {
+    border: none;
+  }
+
+  .languajes:hover .arrows p {
+    transform: rotate(90deg);
+  }
+
+  .am {
+    margin: 0;
+    width: 447px;
+    height: 33px;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 130%;
+    display: flex;
+    align-items: center;
+    color: #091e42;
+  }
+
+  .intro {
+    width: 300px;
+    height: 170px;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .lc {
+    width: 117px;
+    height: 24px;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    color: #7f8cff;
+    margin-bottom: 24px;
+  }
+
+  .icons {
+    display: flex;
+    font-size: 20px;
+  }
+
+  .icons div {
+    padding: 0 10px 0 10px;
+  }
+
+  .resume {
+    width: 158px;
+    height: 49px;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    justify-content: center;
+    letter-spacing: 0.03em;
+    color: #396df2;
+    background-color: inherit;
+    border-radius: 10px;
+    border-color: #396df2;
+    margin-top: 30px;
+  }
+
+  .resume:hover {
+    color: #091e42;
+    border-color: #091e42;
+    background-color: #396df2;
+  }
```

Comment:
Great job going beyond the project requirement and implementing button hover interaction 🎖️